core/jazz_deployments-event-handler/components/utils.js

```diff
@@ -0,0 +1,113 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+    Helper functions for Events
+    @module: utils.js
+    @author:
+    @version: 1.0
+**/
+
+const logger = require("../components/logger.js");
+
+function checkForInterestedEvents(encodedPayload, sequenceNumber, configData) {
+  return new Promise((resolve, reject) => {
+
+    let kinesisPayload = JSON.parse(new Buffer(encodedPayload, 'base64').toString('ascii'));
+    if (kinesisPayload.Item.EVENT_TYPE && kinesisPayload.Item.EVENT_TYPE.S) {
+      if (configData.EVENTS.event_type.includes(kinesisPayload.Item.EVENT_TYPE.S) &&
+          configData.EVENTS.event_name.includes(kinesisPayload.Item.EVENT_NAME.S)) {
+        logger.info("found " + kinesisPayload.Item.EVENT_TYPE.S + " event with sequence number: " + sequenceNumber);
+
+        return resolve({
+          "interested_event": true,
+          "payload": kinesisPayload.Item
+        });
+      } else {
+        logger.error("Not an interested event or event type");
+        return resolve({
+          "interested_event": false,
+          "payload": kinesisPayload.Item
+        });
+      }
+    }
+  });
+};
+
+function getDeploymentPayload(svcContext) {
+  let deploymentPayload = {};
+  svcContext.domain ? deploymentPayload.domain = svcContext.domain : null;
```

Comment:
better way which is more readable:
const deploymentPayload = {
      domain : svcContext.domain ?  svcContext.domain : null,
    .......
}
what do you think?

Reply:
you are right